File: Mcp-Servers/Codebase.Analyzer.Mcp/test-fixtures/god-classes/dotnet/GodOrderService.cs
namespace GodClassFixture;

public class Dep01 { public int V = 1; }
public class Dep02 { public int V = 2; }
public class Dep03 { public int V = 3; }
public class Dep04 { public int V = 4; }
public class Dep05 { public int V = 5; }
public class Dep06 { public int V = 6; }
public class Dep07 { public int V = 7; }
public class Dep08 { public int V = 8; }
public class Dep09 { public int V = 9; }
public class Dep10 { public int V = 10; }
public class Dep11 { public int V = 11; }
public class Dep12 { public int V = 12; }

public class GodOrderService
{
    private int f01, f02, f03, f04, f05, f06, f07, f08, f09, f10;
    private int f11, f12, f13, f14, f15, f16, f17, f18, f19, f20;
    private int f21, f22, f23, f24, f25, f26, f27, f28, f29, f30;

    private readonly Dep01 _d01;
    private readonly Dep02 _d02;
    private readonly Dep03 _d03;
    private readonly Dep04 _d04;
    private readonly Dep05 _d05;
    private readonly Dep06 _d06;
    private readonly Dep07 _d07;
    private readonly Dep08 _d08;
    private readonly Dep09 _d09;
    private readonly Dep10 _d10;
    private readonly Dep11 _d11;
    private readonly Dep12 _d12;

    public GodOrderService(
        Dep01 d01, Dep02 d02, Dep03 d03, Dep04 d04, Dep05 d05, Dep06 d06,
        Dep07 d07, Dep08 d08, Dep09 d09, Dep10 d10, Dep11 d11, Dep12 d12)
    {
        _d01 = d01; _d02 = d02; _d03 = d03; _d04 = d04; _d05 = d05; _d06 = d06;
        _d07 = d07; _d08 = d08; _d09 = d09; _d10 = d10; _d11 = d11; _d12 = d12;
    }

    public int Process01() { f01++; return _d01.V; }
    public int Process02() { f02++; return _d02.V; }
    public int Process03() { f03++; return _d03.V; }
    public int Process04() { f04++; return _d04.V; }
    public int Process05() { f05++; return _d05.V; }
    public int Process06() { f06++; return _d06.V; }
    public int Process07() { f07++; return _d07.V; }
    public int Process08() { f08++; return _d08.V; }
    public int Process09() { f09++; return _d09.V; }
    public int Process10() { f10++; return _d10.V; }
    public int Process11() { f11++; return _d11.V; }
    public int Process12() { f12++; return _d12.V; }
    public int Process13() { f13++; return f13; }
    public int Process14() { f14++; return f14; }
    public int Process15() { f15++; return f15; }
    public int Process16() { f16++; return f16; }
    public int Process17() { f17++; return f17; }
    public int Process18() { f18++; return f18; }
    public int Process19() { f19++; return f19; }
    public int Process20() { f20++; return f20; }
    public int Process21() { f21++; return f21; }
    public int Process22() { f22++; return f22; }
    public int Process23() { f23++; return f23; }
    public int Process24() { f24++; return f24; }
    public int Process25() { f25++; return f25; }
    public int Process26() { f26++; return f26; }
    public int Process27() { f27++; return f27; }
    public int Process28() { f28++; return f28; }
    public int Process29() { f29++; return f29; }
    public int Process30() { f30++; return f30; }
}

File: Claude-Code-Ai/Mcp-Servers/Codebase.Analyzer.Mcp/test-fixtures/god-classes/angular/god-order.service.ts
// Fixture: intentional god class for detect_god_classes assertions.

export class Dep01 { v = 1; }
export class Dep02 { v = 2; }
export class Dep03 { v = 3; }
export class Dep04 { v = 4; }
export class Dep05 { v = 5; }
export class Dep06 { v = 6; }
export class Dep07 { v = 7; }
export class Dep08 { v = 8; }
export class Dep09 { v = 9; }
export class Dep10 { v = 10; }
export class Dep11 { v = 11; }
export class Dep12 { v = 12; }

export class GodOrderService {
  private f01 = 0;
  private f02 = 0;
  private f03 = 0;
  private f04 = 0;
  private f05 = 0;
  private f06 = 0;
  private f07 = 0;
  private f08 = 0;
  private f09 = 0;
  private f10 = 0;
  private f11 = 0;
  private f12 = 0;
  private f13 = 0;
  private f14 = 0;
  private f15 = 0;
  private f16 = 0;
  private f17 = 0;
  private f18 = 0;
  private f19 = 0;
  private f20 = 0;
  private f21 = 0;
  private f22 = 0;
  private f23 = 0;
  private f24 = 0;
  private f25 = 0;
  private f26 = 0;
  private f27 = 0;
  private f28 = 0;
  private f29 = 0;
  private f30 = 0;

  constructor(
    private readonly d01: Dep01,
    private readonly d02: Dep02,
    private readonly d03: Dep03,
    private readonly d04: Dep04,
    private readonly d05: Dep05,
    private readonly d06: Dep06,
    private readonly d07: Dep07,
    private readonly d08: Dep08,
    private readonly d09: Dep09,
    private readonly d10: Dep10,
    private readonly d11: Dep11,
    private readonly d12: Dep12,
  ) {}

  process01() { this.f01++; return this.d01.v; }
  process02() { this.f02++; return this.d02.v; }
  process03() { this.f03++; return this.d03.v; }
  process04() { this.f04++; return this.d04.v; }
  process05() { this.f05++; return this.d05.v; }
  process06() { this.f06++; return this.d06.v; }
  process07() { this.f07++; return this.d07.v; }
  process08() { this.f08++; return this.d08.v; }
  process09() { this.f09++; return this.d09.v; }
  process10() { this.f10++; return this.d10.v; }
  process11() { this.f11++; return this.d11.v; }
  process12() { this.f12++; return this.d12.v; }
  process13() { this.f13++; return this.f13; }
  process14() { this.f14++; return this.f14; }
  process15() { this.f15++; return this.f15; }
  process16() { this.f16++; return this.f16; }
  process17() { this.f17++; return this.f17; }
  process18() { this.f18++; return this.f18; }
  process19() { this.f19++; return this.f19; }
  process20() { this.f20++; return this.f20; }
  process21() { this.f21++; return this.f21; }
  process22() { this.f22++; return this.f22; }
  process23() { this.f23++; return this.f23; }
  process24() { this.f24++; return this.f24; }
  process25() { this.f25++; return this.f25; }
  process26() { this.f26++; return this.f26; }
  process27() { this.f27++; return this.f27; }
  process28() { this.f28++; return this.f28; }
  process29() { this.f29++; return this.f29; }
  process30() { this.f30++; return this.f30; }
}
